tests/Forc.toml

```
@@ -0,0 +1,9 @@
+[project]
+author = "Nicholas Furfaro"
```

Replace in other files too pls
```suggestion
author = "Fuel Labs <contact@fuel.sh>"
```